### DISCUSSION TITLE
Quality Profile Recyclarr Settings Import

### DISCUSSION THREAD

**Discussion by outerepic** [RandomNinjaAtk/arr-scripts#134] [Q&A] [answered]
Hello, I just added these scripts to my *arr setup and had a question. All of the Custom Formats from TRaSH Guides appear to have imported correctly, however the Quality Profiles for Sonarr and Radarr are unchanged, the previous Custom Formats I created are the only ones with a score. Is this expected behavior? 

On the arr-scripts Radarr Read Me it says: "Recyclarr built-in: Auto configures Custom Formats, Auto configures Custom Format Scores." Do I still have to decide which Quality Profiles I want and score them using the imported Custom Formats myself? 

I thought I found the issue while writing this; in the recyclarr.yaml file the API Key imported correctly, but the Base URL was left as the default (http://127.0.0.1:7878) which does not work in my setup (any idea why?). I had to switch the Base URL to http://192.168.XXX.XXX:7878. In my *arr General Settings the Application URL and URL Base are empty.

After two container restarts and a recreate, the Quality Profiles for Radarr and Sonarr are unchanged. Any help would be appreciated!

**Comment by outerepic** [comment, marked as answer]
Updating the recyclarr.yaml files with the templates provided on their wiki solves this

**Reply by RandomNinjaAtk** [reply]
That is correct, the yaml's are designed to be modified to match your taste/preference....

Edit:
The base URL issue you mentioned is likely caused by a user-configuration on your end somewhere.... I've not experienced that.... And because everyone's setups can be different, that is likely the cause...
